apps/i18n/studio/en_us.json

```diff
@@ -285,6 +285,7 @@
   "playSoundWinPoint": "play win point sound",
   "playSoundWinPoint2": "play win point 2 sound",
   "playSoundWood": "play wood sound",
+  "playSoundR2D2Random": "play R2-D2 random sound",
```

Comment:
Written this way instead of "play random R2-D2 sound" so that it would sort nicely in the dropdown?